examples/shell/nxp/k32w/k32w0/main/main.cpp

```diff
@@ -134,7 +134,7 @@ extern "C" void main_task(void const * argument)
     }
 
     // cmd_otcli_init();
-    cmd_ping_init();
+    // cmd_ping_init();
```

Comment:
Should we delete commented out code instead of keeping it in history?

Reply:
Doesn't make any difference to me.